list-pr-files/action.yml

```diff
@@ -0,0 +1,13 @@
+name: "List PR files"
+description: "List pull request files"
+inputs:
+  ghToken:
```

Comment:
```suggestion
  gitHubToken:
```

Reply:
Renamed to `token` which is consistent with `actions/checkout` and others